docs/registry/entities/workflow.md

```diff
@@ -0,0 +1,181 @@
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/entity_namespace.md.j2 -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+# Workflow
```

Comment:
I am missing attributes related to batch processing items. Especially when it comes to metrics, attributes and/or dimensions like `workflow.batch.size`, `workflow.batch.error_count` and `workflow.batch.skip_count` would be helpful. Though given the implied relationship of workflows and tasks, I am unsure where to put the batch information. It could be on either level, depending on whether the workflow iterates over items or tasks first.